sdk/communication/azure-communication-rooms/README.md

```diff
@@ -128,11 +129,12 @@ public void getRoomWithRoomId() {
 ```
 
 ### Delete an existing room
-Use the `deleteRoomWithResponse`  function to delete an existing Room on Azure Communication Service.
+Use the `deleteRoomWithResponse` function to delete an existing Room on Azure Communication Service.
```

Comment:
This was in the original readme, I didn't change it since withReponse there is a way for user to tell if room is deleted or not. I changed it to without response.